### PR TITLE
update local-up-cluster to allow full authentication proxying

### PR DESCRIPTION
**Pull request by deads2k**
Adds group and header information in auth proxy authenticator options for `local-up-cluster.sh`.  Must have been missed in the rebase madness.

### CODE REVIEW

File: hack/local-up-cluster.sh [hunk 1]
```diff
@@ -476,6 +469,11 @@ EOF
       --feature-gates="${FEATURE_GATES}" \
       --cloud-provider="${CLOUD_PROVIDER}" \
       --cloud-config="${CLOUD_CONFIG}" \
+      --requestheader-username-headers=X-Remote-User \
```
Comment (deads2k) [inline hack/local-up-cluster.sh]: @sttts any reason not to do this.  It makes us compatible with `kubernetes-discovery`.

**Comment by deads2k** [comment]
@k8s-bot gce etcd3 e2e test this

**Comment by deads2k** [comment]
@k8s-bot cvm gce e2e test this

**Comment by deads2k** [comment]
@eparis quick look?  It's small.

File: hack/local-up-cluster.sh (hunk 1 above)
Comment (sttts) [inline hack/local-up-cluster.sh]: Don't we have to do this to pass through the user and groups?

Comment (deads2k) [inline hack/local-up-cluster.sh]: > Don't we have to do this to pass through the user and groups?

Yeah, just want to make sure you can't think of a reason.  tag if we're good?

Comment (sttts) [inline hack/local-up-cluster.sh]: Should be fine for the non-proxy case.

**Comment by sttts** [comment]
lgtm

**Pull request merged**